src/frontend/apps/standalone_site/src/features/PagesApi/components/PagesApi.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box } from 'grommet';
+import { normalizeColor } from 'grommet/utils';
+import { theme } from 'lib-common';
+import ReactMarkdown from 'react-markdown';
+import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { ContentSpinner } from 'components/Spinner';
+
+import { usePageApi } from '../api/usePageApi';
+
+const BoxMarkdown = styled(Box)`
+  & p {
+    color: #000;
+  }
+  & a {
+    color: ${normalizeColor('blue-active', theme)};
+  }
+  box-shadow: inset rgba(2, 117, 180, 0.3) 1px 1px 6px 0px;
+`;
+
+const PagesApi = () => {
+  const location = useLocation();
+  const { data, isLoading } = usePageApi(location.pathname);
+
+  return (
+    <Box margin={{ top: 'auto' }}>
+      {isLoading ? (
+        <ContentSpinner boxProps={{ margin: { vertical: 'large' } }} />
+      ) : (
+        <BoxMarkdown
+          width={{ max: 'large' }}
```

Comment:
I think we should always keep the same width.
If the page only contains small phrases, it looks weird IMHO.
![image](https://user-images.githubusercontent.com/720491/236326480-d95e00e9-b311-47ff-b467-dc855f55d1b2.png)

Reply:
👍 [fixup! ✨(website) feature PagesApi](https://github.com/openfun/marsha/pull/2205/commits/40cf211bb3b76590e9a80b8c57486c2f4b912699)